pennylane/transforms/qcut.py

```diff
@@ -2207,20 +2202,9 @@ def remap_tape_wires(tape: QuantumTape, wires: Sequence) -> QuantumTape:
 
     with QuantumTape() as new_tape:
         for op in copy_ops:
-            new_wires = Wires([wire_map[w] for w in op.wires])
-            op._wires = new_wires
-            apply(op)
+            qml.map_wires(op, wire_map=wire_map, queue=True)
```

Comment:
In a follow-up PR (or this one) we could just deprecate/ delete `remap_tape_wires`.  Don't need this function anymore when we can do:
```
qml.map_wires(tape, dict(zip(tape.wires, wires)))
```

Reply:
Done!